src/store/actions/AnimateMap.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  AnimateMapStageOptions,
+  ReplicatedUser,
+  ReplicatedVenue,
+} from "../reducers/AnimateMap";
+import { ReduxAction } from "../../types/redux";
+
+export enum AnimateMapActionTypes {
+  UPDATE_STAGE_OPTIONS = "AnimateMapActionTypes.UPDATE_STAGE_OPTIONS",
+  UPDATE_ZOOM = "AnimateMapActionTypes.UPDATE_ZOOM",
+  UPDATE_USERS = "AnimateMapActionTypes.UPDATE_USERS",
+  UPDATE_VENUES = "AnimateMapActionTypes.UPDATE_VENUES",
+}
+
+export type setAnimateMapStageOptionsAction = ReduxAction<
+  AnimateMapActionTypes.UPDATE_STAGE_OPTIONS,
+  {
+    options: AnimateMapStageOptions;
+  }
+>;
+
+export type setAnimateMapZoomAction = ReduxAction<
+  AnimateMapActionTypes.UPDATE_ZOOM,
+  {
+    zoom: number;
+  }
+>;
+
+export type updateAnimateMapUsersAction = ReduxAction<
+  AnimateMapActionTypes.UPDATE_USERS,
+  {
+    users: Map<string, ReplicatedUser>;
+  }
+>;
+
+export type updateAnimateMapVenuesAction = ReduxAction<
+  AnimateMapActionTypes.UPDATE_VENUES,
+  {
+    venues: Map<string, ReplicatedVenue>;
+  }
+>;
+
+export const setAnimateMapStageOptions = (
+  options?: AnimateMapStageOptions
+): setAnimateMapStageOptionsAction => ({
+  type: AnimateMapActionTypes.UPDATE_STAGE_OPTIONS,
+  payload: { options },
+});
```

Comment:
The name of these Redux actions should be aligned to their type. So instead of `setAnimateMapStageOptions` this would be `updateAnimateMapStageOptions` or similar.

Reply:
In general though, actions shouldn't be defined as 'get' / 'set' / etc, but should rather be modelled as events. See the Redux styleguide/best practices for more information:

- https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters